mitiq/pec/representations/learning.py

```diff
@@ -84,3 +98,78 @@ def _biased_noise_loss_function(
     return np.sum((mitigated_values - ideal_values) ** 2) / len(
         training_circuits
     )
+
+
+def _unmitigated_loss_function(
+    epsilon_guess: np.ndarray,
+    noisy_executor: Executor,
+    training_circuits: List[QPROGRAM],
+    observable: Optional[Observable] = None,
+) -> float:
+    def ideal_execute(circ: Circuit) -> np.ndarray:
+        return compute_density_matrix(circ, noise_level=(0.0,))
+
+    def noise_model_execute(circ: Circuit) -> np.ndarray:
+        circuit = convert_to_mitiq(circ)[0]
+        noisy_circ = circuit.with_noise(
+            biased_noise_channel(epsilon=epsilon_guess[0], eta=0)
+        )
+        return ideal_execute(noisy_circ)
+
+    noise_model_executor = Executor(noise_model_execute)
+    noise_model_values = np.array(
+        [
+            noise_model_executor.evaluate(training_circuit, observable)
+            for training_circuit in training_circuits
+        ]
+    )
+
+    noisy_values = np.array(
+        [
+            noisy_executor.evaluate(training_circuit, observable)
+            for training_circuit in training_circuits
+        ]
+    )
+
+    return np.sum(
+        abs(noise_model_values.reshape(-1, 1) - noisy_values.reshape(-1, 1))
+        ** 2
+    ) / len(training_circuits)
+
+
+def biased_noise_channel(epsilon: float, eta: float) -> MixedUnitaryChannel:
+    a = 1 - epsilon
+    b = epsilon * (3 * eta + 1) / (3 * (eta + 1))
+    c = epsilon / (3 * (eta + 1))
+
+    mix = [
+        (a, unitary(I)),
+        (b, unitary(Z)),
+        (c, unitary(X)),
+        (c, unitary(Y)),
+    ]
+    return ops.MixedUnitaryChannel(mix)
+
+
+def learn_noise_parameters_from_unmitigated_data(
+    circuit: QPROGRAM,
+    epsilon0: float,
+    noisy_executor: Executor,
+    num_training_circuits: int = 5,
+    fraction_non_clifford: float = 0.2,
+    training_random_state: np.random.RandomState = None,
+    observable: Optional[Observable] = None,
```

Comment:
This function could take `operations_to_learn` as an input, as we did for the PEC learning function.
In that case, `_unmitigated_loss_function` should simulate noise only after the operations in `operations_to_learn` .